cli/mender-artifact/read.go

```diff
@@ -81,6 +81,7 @@ func readArtifact(c *cli.Context) error {
 	err = ar.ReadArtifact()
 	if err != nil {
 		return cli.NewExitError(err.Error(), 0)
+		return cli.NewExitError(err.Error(), 1)
```

Comment:
I take it you wanted to remove the other one? 🙂

Reply:
I will fix this  up last, so that I don't have to force push during review.